crates/action-graph/src/action_graph.rs

```diff
@@ -23,7 +23,7 @@ impl ActionGraph {
         ActionGraph { graph }
     }
 
-    pub fn creater_iter(&self, indices: Vec<NodeIndex>) -> ActionGraphIter {
+    pub fn create_iter(&self, indices: Vec<NodeIndex>) -> ActionGraphIter {
```

Comment:
this should probably require a major release?

I can revert it if that is undesirable?

Reply:
We don't publish these crates so it's fine.